engine-standalone-storage/src/lib.rs

```diff
@@ -198,7 +198,9 @@ impl Storage {
         let storage_key = construct_storage_key(StoragePrefix::Diff, &tx_included.to_bytes());
         self.db
             .get_pinned(storage_key)?
-            .map(|slice| Diff::try_from_bytes(slice.as_ref()).unwrap())
+            .map(|slice| {
+                Diff::try_from_bytes(slice.as_ref()).expect("transaction_diff should be valid")
```

Comment:
Let's change `should be valid` => `is invalid`?

Reply:
Addressed in https://github.com/aurora-is-near/aurora-engine/pull/916/commits/28be3e2a84ba8797f1f8838f4e45f1e98e16a1f9